reference/gradle-plugin-configuration.md

```diff
@@ -146,6 +146,14 @@ rewrite {
 }
 ```
 
+If desired, `rewriteDryRun` can be configured so that when `check` runs, `rewriteDryRun` does too:
+
+```gradle
```

Comment:
I don't think "gradle" is an option GitBook markdown supports. "groovy" is probably the best fit.